userns.go

```diff
@@ -356,6 +356,7 @@ func findAvailableIDRange(size uint32, availableIDs, usedIDs []idtools.IDMap) ([
 			return avail[:i+1], nil
 		}
 		remaining -= uint32(avail[i].Size)
+		currentID += avail[i].Size
```

Comment:
I don't think so, but will ask.  Any chance the currentID could get too big here?

Reply:
the sum of all avail[].Size cannot be bigger than 4294967295 and we are counting from 0.

In the worse case, the kernel will reject the user namespace configuration